lib/api/actions/functions.php

```diff
@@ -592,6 +584,45 @@ function _beans_is_action_valid( array $action ) {
 	return isset( $action['hook'], $action['callback'], $action['priority'], $action['args'] );
 }
 
+/**
+ * Build the action's array for only the valid given arguments.
+ *
+ * @since 1.5.0
+ *
+ * @param string|null   $hook     Optional. The action event's name to which the $callback is hooked.
+ *                                Valid when not falsey,
+ *                                i.e. (meaning not `null`, `false`, `0`, `0.0`, an empty string, or empty array).
+ * @param callable|null $callback Optional. The callback (function or method) you wish to be called when the event
+ *                                fires. Valid when not falsey, i.e. (meaning not `null`, `false`, `0`, `0.0`, an empty
+ *                                string, or empty array).
+ * @param int|null      $priority Optional. Used to specify the order in which the functions associated with a
+ *                                particular action are executed. Valid when it's numeric, including 0.
+ * @param int|null      $args     Optional. The number of arguments the callback accepts.
+ *                                Valid when it's numeric, including 0.
+ *
+ * @return array
+ */
+function _beans_build_action_for_valid_args( $hook = null, $callback = null, $priority = null, $args = null ) {
```

Comment:
As I rule of thumb, I am more in favor of keep function name as short as possible, keeping in mind that the long the function name is, the more memory it uses.